versions.properties

```diff
@@ -0,0 +1,54 @@
+## suppress inspection "SpellCheckingInspection" for whole file
+##
+## Dependencies and Plugin versions with their available updates
+## Generated by $ ./gradlew refreshVersions
```

Comment:
What kind of control do we have when there are specific versions we don't want to bump? I guess we just don't use the `_`?

Reply:
Yep. But the tooling is very safe – you run `./gradlew refreshVersions`, and it just adds a bunch of comments to that file telling you what the latest versions are. You have to actually edit it to update.